plugins/out_s3/s3_multipart.c

```diff
@@ -431,6 +431,10 @@ int complete_multipart_upload(struct flb_s3 *ctx,
     struct flb_http_client *c = NULL;
     struct flb_aws_client *s3_client;
 
+    if (!m_upload->upload_id) {
+        return -1;
+    }
```

Comment:
let's add a log message just to give some record of why this function failed? this will help in any issues that users open

Reply:
I've added one, let me know if it's not an appropriate format.